src/jaxsim/api/contact.py

```diff
@@ -269,3 +274,124 @@ def estimate_model_height(model: js.model.JaxSimModel) -> jtp.Float:
     )
 
     return sc_parameters
+
+
+@jax.jit
+def transforms(model: js.model.JaxSimModel, data: js.data.JaxSimModelData) -> jtp.Array:
+    r"""
+    Return the pose of the collidable points.
```

Comment:
Can we be a little bit more verbose?

```suggestion
    Return the pose of the collidable points in inertial-fixed representation.
```

Reply:
The $\text{SE}(3)$ pose does not depend on any velocity representation. In JaxSim, poses are always absolute, i.e. w.r.t. the world frame $W$.